contracts/src/v0.8/automation/ExecutionPrevention.sol

```diff
@@ -8,8 +8,8 @@ abstract contract ExecutionPrevention {
    * @notice method that allows it to be simulated via eth_call by checking that
    * the sender is the zero address.
    */
-  function preventExecution() internal view {
-    if (tx.origin != address(0)) {
+  function _preventExecution() internal view {
+    if (msg.sender != address(0)) {
```

Comment:
i think this needs to remain tx.origin

Reply:
Gotcha, thanks @FelixFan1992 for confirming, will update now!